# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ron/tests/465_unwrap_some_newtype_variant_value.rs
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 third_party/rust/ron/tests/465_unwrap_some_newtype_variant_value.rs
--- a/third_party/rust/ron/tests/465_unwrap_some_newtype_variant_value.rs	Fri Oct 03 18:01:30 2025 +0000
+++ b/third_party/rust/ron/tests/465_unwrap_some_newtype_variant_value.rs	Fri Oct 03 18:08:29 2025 +0000
@@ -4,7 +4,10 @@
         ron::from_str::<ron::Value>("Some(a: 42)"),
         Err(ron::error::SpannedError {
             code: ron::Error::ExpectedOptionEnd,
-            position: ron::error::Position { line: 1, col: 7 },
+            span: ron::error::Span {
+                start: ron::error::Position { line: 1, col: 7 },
+                end: ron::error::Position { line: 1, col: 7 },
+            }
         }),
     );
     assert_eq!(